src/Analysis/Ast/Impl/Types/PythonFunctionType.cs

```diff
@@ -25,6 +25,7 @@
 namespace Microsoft.Python.Analysis.Types {
     [DebuggerDisplay("Function {Name} ({TypeId})")]
     internal sealed class PythonFunctionType : PythonType, IPythonFunctionType {
+        private readonly IReadOnlyList<string> DefaultClassMethods = new[] { "__new__", "__init_subclass__", "__class_getitem__" };
```

Comment:
Hashset is probably better for `Contains`

Reply:
Yeah, with 3 items, an array is almost definitely going to be faster.

Reply:
It should however be a static member; it doesn't need to be allocated for every `PythonFunctionType`.

Reply:
OK